h/static/styles/about-this-version-dialog.scss

```diff
@@ -1,23 +1,49 @@
+.about-this-version-dialog {
```

Comment:
By my understanding of the BEM pattern for CSS naming, shouldn't this top one be called `.about-this-version`, and then the various components be things like `.about-this-version__title`, `.about-this-version__content`?

Perhaps I have the wrong end of the stick. It wouldn't be the first time. CSS not my forte.